backend/tests_configs.ini

```diff
@@ -456,8 +492,14 @@ user.reset_password.token_lifetime = 5
 
 email.notification.share_content_to_emitter.template.html = %(here)s/tracim_backend/templates/mail/shared_content_to_emitter_body_html.mak
 email.notification.share_content_to_receiver.template.html = %(here)s/tracim_backend/templates/mail/shared_content_to_receiver_body_html.mak
+email.notification.upload_permission_to_emitter.template.html = %(here)s/tracim_backend/templates/mail/upload_permission_to_emitter_body_html.mak
+email.notification.upload_permission_to_receiver.template.html = %(here)s/tracim_backend/templates/mail/upload_permission_to_receiver_body_html.mak
+email.notification.new_upload_event.template.html = %(here)s/tracim_backend/templates/mail/new_upload_event_body_html.mak
+
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
 email.notification.share_content_to_receiver.subject = [{website_title}] {emitter_name} shared content {content_filename} with you
+email.notification.upload_permission_to_emitter.subject = [{website_title}] you give upload permission on {workspace_name} to {nb_receivers} users
+email.notification.upload_permission_to_receiver.subject = [{website_title}] {emitter_name} give you access to upload content into {workspace_name} share space
```

Comment:
I see these configs are required: they are added on all configs. Maybe these configs can be optional + default value ?
Related issue (dot not change anything yet): #2018

Reply:
These config are optional (like other .subject), this is just value used in test.

Reply:
Template config aren't optional and can't be, because they need absolute path. To support those with default value, we probably do need some hack, not sure how to do this.

Reply:
Ok. Does it exist an issue about absolute path problematic as you say ? If not create one. This is a technical debt.

Reply:
issue related there #2204